root/static/scripts/release-editor/edits.js

```diff
@@ -496,20 +496,20 @@ const ERROR_NO_CHANGES = 3;
 
             let submissionDone = function (data) {
                 if (data && current.callback) {
-                    current.callback(release, data.edits);
+                    current.callback(
+                        release,
+                        data.edits.filter(
+                            x => x.response === WS_EDIT_RESPONSE_OK
```

Comment:
The callbacks in `releaseEditor.orderedEditSubmissions` are only written to handle edits that actually went through. But, this technically isn't needed, because "create" edits (which are mostly what we need to handle in those callbacks) can't trigger NoChanges exceptions.